media-libs/libfdk_aac/libfdk_aac-0.1.5.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="Fraunhofer AAC codec library"
+DESCRIPTION='The Fraunhofer FDK AAC Codec Library for Android ("FDK AAC Codec") \
+	is software that implements the MPEG Advanced Audio Coding ("AAC") \
+	encoding and decoding scheme for digital audio.'
+HOMEPAGE="https://sourceforge.net/projects/opencore-amr"
+COPYRIGHT="1995-2012 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e.V."
+LICENSE="Fraunhofer FDK AAC Codec"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/opencore-amr/fdk-aac/fdk-aac-$portVersion.tar.gz"
+CHECKSUM_SHA256="2164592a67b467e5b20fdcdaf5bd4c50685199067391c6fcad4fa5521c9b4dd7"
+SOURCE_DIR="fdk-aac-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
```

Comment:
Shall we mark it as tested on x86 secondary arch? (It builds fine for me on that arch.)
```diff
-SECONDARY_ARCHITECTURES="?x86"
+SECONDARY_ARCHITECTURES="x86"
```

Reply:
I've already built it myself, but forgot to commit that change :P